docs/vm-setup.md

```diff
@@ -26,6 +26,13 @@
     2. Do `bcdedit.exe -set TESTSIGNING ON`.
     3. Restart the VM so that the change will be applied.
 
+4. Install the matching version of the Microsoft Visual C++ Runtime:
+Versions of the Microsoft C Runtime are available on the developer machine, located in %ProgramFiles(x86)%\Microsoft Visual Studio\2019\Enterprise\VC\Redist\MSVC\version, where version is updated with each patch of Visual Studio.
```

Comment:
No such path on my machine.  Specifically we do NOT require Enterprise edition.  Community edition should work.